pip/requirements.txt

```diff
@@ -1,16 +1,31 @@
-Pygments==2.7.4
-bleach==3.3.0
-certifi==2018.11.29
-chardet==3.0.4
-docutils==0.14
-idna==2.8
-pkginfo==1.5.0.1
-readme-renderer==24.0
-requests-toolbelt==0.8.0
-requests==2.27.0
-setuptools==62.1.0
-tqdm==4.29.1
-twine==1.12.1
-urllib3==1.26.5
+bleach==6.0.0
```

Comment:
Also, we notice you've updated a lot of dependencies. We try not to break compatibility without reason.

Reply:
Looks like the repo didn't implement an interpreter solution for those rules. So although there is a requirements.txt , those deps are actually not used correctly. Based on the implementation the rules will use the system  dependency rather than what it installed in the sandbox through requirements.txt . I changed a bit in `deploy.py`  to make it use the runfile dependency first to avoid the issue.  The change is included in `fix dependency issue` commit.

The reason there are many deps missing in the original requirements.txt is because the rule is using your system packages.

Reply:
Good heavens, I didn't notice that little change. Thanks!